auth-api/requirements.txt

```diff
@@ -1,65 +1,69 @@
-Flask-Caching==2.0.1
+Flask-Caching==2.0.2
 Flask-Mail==0.9.1
 Flask-Migrate==2.7.0
 Flask-Moment==1.0.5
 Flask-SQLAlchemy==2.5.1
 Flask-Script==2.0.6
 Flask==1.1.4
 Jinja2==2.11.3
-Mako==1.2.3
+Mako==1.2.4
 MarkupSafe==2.0.1
-SQLAlchemy-Continuum==1.3.13
-SQLAlchemy-Utils==0.38.3
+SQLAlchemy-Continuum==1.3.14
+SQLAlchemy-Utils==0.40.0
 SQLAlchemy==1.3.24
 Werkzeug==1.0.1
-alembic==1.8.1
+alembic==1.10.0
 aniso8601==9.0.1
 asyncio-nats-client==0.11.5
 asyncio-nats-streaming==0.4.0
-attrs==22.1.0
+attrs==22.2.0
 bcrypt==4.0.1
 blinker==1.5
 cachelib==0.9.0
 certifi==2022.12.7
-charset-normalizer==2.1.1
+charset-normalizer==3.1.0
 click==7.1.2
 ecdsa==0.18.0
 expiringdict==1.2.2
 flask-jwt-oidc==0.3.0
 flask-marshmallow==0.11.0
-flask-restx==0.5.1
+flask-restx==1.1.0
```

Comment:
I noticed this is quite an upgrade, they resumed the project.. might want to be careful with this one though

Reply:
yeah I'll remove all these for this PR, but we should regularly upgrade the reqs (probably once per sprint). Do you think it would be good to add a ticket for this every time with 1 estimate?

That way we don't gradually fall behind

Reply:
I wouldn't worry about it, I usually update them